### DISCUSSION TITLE
ARM64 Support

### DISCUSSION THREAD

**Discussion by amplitudesxd** [postalserver/postal#2768] [Feature suggestions]
Hi,

I currently have a fork of Postal which I created (https://github.com/ClyentSoftwares/postal) for ARM64 support, as well as fixing a few other bugs.

Would an ARM64 build be something that could be implemented in the main project? It's as simple as adding `platforms: linux/arm64/v8,linux/amd64` to the GitHub workflow on Docker build steps.

Happy to open a PR if so, but unsure as I've seen a previous issue about ARM support just being closed.

**Comment by willpower232** [comment]
I don't believe it has been explicitly rejected, I'm sure a PR would be appreciated to bring more attention to how much effort it requires, even if that effort is none!

**Reply by amplitudesxd** [reply]
Thank you for your response! In that case, I've opened a PR: https://github.com/postalserver/postal/pull/2770

**Reply by casualmatt** [reply]
@amplitudesxd I looked at your fork, and the spam header is a nice feature; you should also open a PR for that.
